docs/source/examples/managed-jobs.rst

```diff
@@ -288,29 +288,18 @@ will be spot instances. If spot instances are not available, SkyPilot will fall
 Checkpointing and recovery
 --------------------------
 
-To recover quickly, a cloud bucket is typically needed to store the job's states (e.g., model checkpoints). Any data on disk that is not stored inside a cloud bucket will be lost during the recovery process.
+To recover quickly from spot instance preemptions, a cloud bucket is typically needed to store the job's states (e.g., model checkpoints). Any data on disk that is not stored inside a cloud bucket will be lost during the recovery process.
 
-Below is an example of mounting a bucket to :code:`/checkpoint`.
+Below is an example of mounting a bucket to :code:`/checkpoint`:
 
 .. code-block:: yaml
 
   file_mounts:
     /checkpoint:
       name: # NOTE: Fill in your bucket name
-      mode: MOUNT
-
-The :code:`MOUNT` mode in :ref:`SkyPilot bucket mounting <sky-storage>` ensures the checkpoints outputted to :code:`/checkpoint` are automatically synced to a persistent bucket.
-
-To implement checkpointing in your application code:
-
-1. Periodically save checkpoints containing the current program state during execution.
-2. When starting/restarting, check if any checkpoints are present, and load the latest checkpoint.
-
-Both features are included in most model training libraries, such as `PyTorch <https://pytorch.org/docs/stable/checkpoint.html>`__, `TensorFlow <https://www.tensorflow.org/guide/checkpoint>`__, and `Hugging Face <https://huggingface.co/docs/transformers/main_classes/trainer#transformers.TrainingArguments.save_steps>`__.
```

Comment:
People reading this section may have the question: "How do I actually set up the checkpointing?" Can we keep some info about this? If we want to just link to high-performance training page, we can be clear that it also has details on how to set up the checkpointing.

Reply:
This is still outstanding.